newsletter-examples/NOAA-CRW-DHW.ipynb

```diff
@@ -0,0 +1,288 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Extraction of a time series of Degree Heating Weeks (DHW) from the NOAA CRW collection\n",
+    "\n",
+    "*This is the notebook used to create the timeseries plot showcased in the DMS Newsletter #2*\n",
+    "\n",
+    "If you want to try this notebook using NECTAR's JupyterHub service just click here (you need to have an academic email registered with AAF): [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ausseabed/dms-newsletter-2/HEAD?filepath=notebooks%2FDHW_timeseries.ipynb)\n",
+    "\n",
+    "What you need to know:\n",
+    "- the location of the Great Barrier Reef you want to assess. It must be a single point (lat, lon)\n",
+    "- the time period you want to assess. It must be a range of years (e.g. 2010-2019)\n",
+    "- the name of the output file you want to create. It must be a string (e.g. 'my_output_file.csv')\n",
```

Comment:
What's the output CSV used for? There are no codes in this notebook to generate CSV.

Reply:
keep as it will be used at the end